vehicle/renault.go

```diff
@@ -163,11 +163,16 @@ func (v *Renault) apiKeys(region string) error {
 
 	var cr configResponse
 	err := v.GetJSON(uri, &cr)
-
-	v.gigya = cr.Servers.GigyaProd
-	v.kamereon = cr.Servers.WiredProd
-
-	return err
+	if err != nil {
+		// Use of old fixed keys if keyStore is not accessible
+		v.gigya = configServer{"https://accounts.eu1.gigya.com", "3_7PLksOyBRkHv126x5WhHb-5pqC1qFR8pQjxSeLB6nhAnPERTUlwnYoznHSxwX668"}
+		v.kamereon = configServer{"https://api-wired-prod-1-euw1.wrd-aws.com", "VAX7XYKGfa92yMvXculCkEFyfZbuM7Ss"}
+		return nil
+	} else {
```

Comment:
Das kannst du einfach ohne else Zweig schreiben.

Reply:
Wollen wir die Hoffnung direkt aufgeben, dass es keinen Ersatz für die keyStore URL gibt?

Reply:
Wenn man das wüsste...
Vorschlag: Lass uns mal noch ein bisschen die Lage beobachten.

Reply:
Der Kamereon-API-Key war ja im Keystore auch schon veraltet und musste gepatched werden...

Frage: Was haltet ihr von Verschieben der 4 keyStore Parameter in die evcc.yaml und entsprechenden default Werten im renault Vehicle configure Template ...

Reply:
Fände ich eine gute Lösung. Dann könnte man sich notfalls auch selbst Abhilfe schaffen ohne auf ein neues Release warten zu müssen.